README.md

```diff
@@ -72,26 +72,23 @@ Then run:
 * [Install pre-commit globally](https://pre-commit.com/#installation)
 * From your checkout directory run `pre-commit install`
 
-## Run locally
+## Run App or Unit Tests Locally
+Local Docker stack must be up-to-date and running with `docker compose up` in order to use the app/tests locally:
+https://github.com/communitiesuk/funding-service-design-post-award-docker-runner
+
+Make migrations to ensure Docker PostgreSQL container is up-to-date:
+
 `flask db migrate`
+
```

Comment:
after running `flask db migrate` on an fresh database container _**with**_ data-store and account-store databases created

I get the error:
```
INFO  [alembic.runtime.migration] Context impl PostgresqlImpl.
INFO  [alembic.runtime.migration] Will assume transactional DDL.
ERROR [flask_migrate] Error: Target database is not up to date.
```

and have to run `flask db upgrade` to apply all existing migrations. Should this actually say `flask db upgrade`?

Reply:
after running `flask db migrate` on an fresh database container _**without**_ data-store and account-store databases created

I get the error:
```
sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) connection to server at "localhost" (::1), port 5432 failed: FATAL:  database "data_store" does not exist
```

I know how to fix this because I know that the docker-compose can run into race conditions and not create these but should we include some information for a reader that may find this confusing?

Reply:
yeah, we should. I'd kind of assumed that they would just read the README on the compose repo, and was hesitant to replicate that info here, but you're right it will probably be more confusing not to include it here

Reply:
updated the docs